packages/kit/src/exports/vite/index.js

```diff
@@ -379,6 +379,8 @@ function kit() {
 				);
 
 				log.info('Building server');
+				const assets_prefix = `${svelte_config.kit.paths.base}/${svelte_config.kit.appDir}/`;
```

Comment:
it looks to me like this should not have a trailing slash or you'll end up with `//` in the final path

Reply:
and it also looks like you're going to end up with at least a `//` at the beginning because `base` must start with `/` and then you add it to a string starting with `/`

Reply:
The `/` between the two paths is now dependent upon `paths.base` not ending in `/`